Library/Homebrew/brew.sh

```diff
@@ -129,6 +129,11 @@ else
   HOMEBREW_CURL="curl"
 fi
 
+if [[ -n "$HOMEBREW_DEVELOPER" && -x "$HOMEBREW_CURL_PATH" ]]
```

Comment:
What about putting this as an `elif` above; it feels like if `HOMEBREW_FORCE_BREWED_CURL` is set this shouldn't override it?